tidysic/audio_file.py

```diff
@@ -86,35 +86,47 @@ def read_tags(self):
             for tag in Tag
         }
 
-    def save_tags(self, new_tags, dry_run):
+    def save_tags(
+        self,
+        verbose: bool = False,
+        dry_run: bool = False
+    ):
         '''
-        Applies the given collection of tags to itself and
-        saves them to the file (if `dry_run == True`)
+        Saves the current tag values to the file.
+
+        Args:
+            verbose (bool): If True, each tag saving will be displayed.
+            dry_run (bool): If True, the tags won't actually be saved, but the
+                would-be modifications are displayed.
         '''
-        for tag, value in new_tags.items():
-            self.tags[tag] = value
 
+        old_tags = self.read_tags()
+
+        try:
```

Comment:
The 4 following lines are a duplicate of [ `read_tags()`](https://github.com/tidysic/tidysic/blob/b3e2cc0dd1dcb228948e9143b016985ede7e12d2/tidysic/audio_file.py#L74), I'm sure we can do better here (DRY)

Reply:
I thought about it. `old_tags`can be read from the `file_tags`instead, but I would have to repeat the weird accessing found in `read_tags()`. I'll try and make it slicker.